sinks.go

```diff
@@ -46,6 +46,20 @@ func (p *Pipe) CountLines() (int, error) {
 	return lines, p.Error()
 }
 
+// Slice return each lines of the pipe in a Slice or an error.
+// If there is an error reading the pipe, the pipe's error status
+// is also set.
+func (p *Pipe) Slice() ([]string, error) {
+	if p == nil || p.Error() != nil {
+		return nil, p.Error()
+	}
+	var slice []string
```

Comment:
We don't usually name variables after their type, as tempting as it is sometimes. How about `result` here instead?